### PR TITLE
180699160 add dao to homepage

### PR DESCRIPTION
**Pull request by kbiala**
#### What's this PR do?
- Adds DAO pool to homepage

#### How should this be manually tested?
- Check homepage carousels on desktop and mobile devices

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/n/projects/2177859/stories/180699160)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?

### CODE REVIEW

File: app/helpers/validators_helper.rb [hunk 1]
```diff
@@ -79,10 +79,11 @@ def solstake_url(vote_key)
...
   def shuffle_logos
     [
-      ['https://lido.fi/solana', 'lido.png'],
-      ['https://marinade.finance', 'marinade.png'],
-      ['https://www.socean.fi', 'socean.png'],
-      ['https://jpool.one/', 'jpool.png']
+      ["https://lido.fi/solana", "lido.png"],
+      ["https://marinade.finance", "marinade.png"],
+      ["https://www.socean.fi", "socean.png"],
+      ["https://jpool.one/", "jpool.png"],
```
Comment (maciejkocylapc) [inline app/helpers/validators_helper.rb:85]:
```suggestion
      ["https://jpool.one", "jpool.png"],
```

File: app/helpers/validators_helper.rb [hunk 2]
```diff
@@ -79,10 +79,11 @@ def solstake_url(vote_key)
...
+      ["https://daopool.monkedao.io/", "daopool.png"]
```
Comment (maciejkocylapc) [inline app/helpers/validators_helper.rb:86]:
```suggestion
      ["https://daopool.monkedao.io", "daopool.png"]
```

File: app/views/validators/_banner.html.erb [hunk 3]
```diff
@@ -39,5 +39,23 @@
...
+            <% end %>
+          </div>
+        <% end %>
+      </div>
+      <ol class="carousel-indicators">
+        <li data-target="#carouselLogosMobile" data-slide-to="0" class="active"></li>
+        <li data-target="#carouselLogosMobile" data-slide-to="1"></li>
+        <li data-target="#carouselLogosMobile" data-slide-to="2"></li>
+        <li data-target="#carouselLogosMobile" data-slide-to="3"></li>
+        <li data-target="#carouselLogosMobile" data-slide-to="4"></li>
```
Comment (maciejkocylapc) [inline app/views/validators/_banner.html.erb:57]: You could use rubys `n.times do ` to write this

**Comment by kbiala** [comment]
@maciejkocylapc Comments applied. Can you take another look?

**Review by maciejkocylapc — APPROVED** [review]

**Pull request merged**